desc/objectives/_generic.py

```diff
@@ -320,4 +331,152 @@ def compute(self, R_lmn, Z_lmn, L_lmn, i_l, c_l, Psi, **kwargs):
         )
         I = compress(self.grid, data["current"], surface_label="rho")
         w = compress(self.grid, self.grid.spacing[:, 0], surface_label="rho")
-        return self._shift_scale(I * w)
+        return self._shift_scale(I) * w
+
+
+class TargetIota(_Objective):
+    """Targets a rotational transform profile.
+
+    Parameters
+    ----------
+    eq : Equilibrium, optional
+        Equilibrium that will be optimized to satisfy the Objective.
+    target : float, ndarray, optional
+        Target value(s) of the objective.
+        len(target) must be equal to Objective.dim_f == grid.num_rho
+    weight : float, ndarray, optional
+        Weighting to apply to the Objective, relative to other Objectives.
+        len(weight) must be equal to Objective.dim_f == grid.num_rho
+    normalize : bool
+        Whether to compute the error in physical units or non-dimensionalize.
+        Note: has no effect for this objective.
+    normalize_target : bool
+        Whether target should be normalized before comparing to computed values.
+        if `normalize` is `True` and the target is in physical units, this should also
+        be set to True.
+        Note: has no effect for this objective.
+    grid : Grid, optional
+        Collocation grid containing the nodes to evaluate at.
+    name : str
+        Name of the objective function.
+    """
+
+    _scalar = False
+    _linear = False
+    _units = "(n/a)"
```

Comment:
In other objectives (like aspect ratio) we say `"(dimensionless)"`. We should be consistent -- which do we prefer?

Reply:
yeah dimensionless is probably better ill change it